soc/nordic/validate_base_addresses.c

```diff
@@ -319,6 +321,8 @@ CHECK_DT_REG(uart00, NRF_UARTE00);
 CHECK_DT_REG(uart20, NRF_UARTE20);
 CHECK_DT_REG(uart21, NRF_UARTE21);
 CHECK_DT_REG(uart22, NRF_UARTE22);
+CHECK_DT_REG(uart23, NRF_UARTE23);
+CHECK_DT_REG(uart24, NRF_UARTE24);
```

Comment:
Unintended change, I guess.

Reply:
> Intended 😄

Well, the commit message does not mention anything about UARTs.

> UART23 ,24 was added in this PR: https://github.com/zephyrproject-rtos/zephyr/pull/90373/files
> 
> Not sure if that was "complete".

The above change does not complete it either. `NRFX_UARTE23/24` Kconfig options and their translations to nrfx configuration macros are still missing.